CRM/Utils/Address/BatchUpdate.php

```diff
@@ -221,6 +226,10 @@ public function processContacts($processGeocode, $parseStreetAddress) {
         // when we should found street_number and street_name
         if (empty($parsedFields['street_name']) || empty($parsedFields['street_number'])) {
           $success = FALSE;
+          // If an address has failed in the geocoding scheduled job i.e. no lat/long is fetched, we will update the manual_geocode field to 1. 
```

Comment:
@maryguo2023 these 4 lines need to be moved in Old line 214

Reply:
the code there should be like the following as this section is about address parsing not geocoding of addresses

```php
        if (isset($params['geo_code_1']) && $params['geo_code_1'] != 'null') {
          $totalGeocoded++;
          $addressParams = $params;
        }
        else {
           $addressParams['manual_geo_code'] = TRUE;
           $addressParams['geo_code_1'] = 0;
           $addressParams['geo_code_2'] = 0;
        }
```